Formula/s/smpeg2.rb

```diff
@@ -1,12 +1,19 @@
 class Smpeg2 < Formula
   desc "SDL MPEG Player Library"
   homepage "https://icculus.org/smpeg/"
-  url "svn://svn.icculus.org/smpeg/tags/release_2_0_0/", revision: "408"
-  head "svn://svn.icculus.org/smpeg/trunk"
+  # license change was done in 2021 Aug, which is 8 years after 2.0.0 release
+  # commit ref, https://github.com/icculus/smpeg/commit/ffa0d54
+  url "https://github.com/icculus/smpeg/archive/refs/tags/release_2_0_0.tar.gz"
```

Comment:
It says 8 years for both 0.4.5 and 2.0.0, are these both correct?

Also if so: why are we on an 8 year old version?

Reply:
mostly because it is buildable and running? (same goes to lots of other formulae)

Reply:
upstream does have some new release chat, https://github.com/icculus/smpeg/issues/9

Reply:
Yeh, that's fine, just checking it was correct here as comment same in both place.

Reply:
haha, yeah, the 2.0.0 and 0.4.5 got released almost side to side.